api/http/server.go

```diff
@@ -867,6 +875,11 @@ func (s *Server) HandleGetList(w http.ResponseWriter, r *http.Request) {
 			respondError(w, r, err.Error(), http.StatusUnauthorized)
 			return
 		}
+		if isRecoveryAttemptError(err) {
+			w.Header().Set("WWW-Authenticate", fmt.Sprintf("Basic realm=%q", uri.Address().String()))
```

Comment:
i think this line should match `874` and it does not

Reply:
You are right, fixed